hack/make-rules/verify.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 # Copyright 2014 The Kubernetes Authors.
+# Copyright 2020 Authors of Arktos - file modified.
```

Comment:
nit: Arktos copyright does not apply here

Reply:
Arktos copyright removed.